package.json

```diff
@@ -81,6 +84,13 @@
       "post-rewrite": "yarn run ts-node ./scripts/hooks/post-rewrite.ts"
     }
   },
+  "jest": {
```

Comment:
Remove, you should add additional jest configuration in the `jest` subfolder. You want to make additional configurations like `test-ci-coverage.js` and `e2e-ci-coverage.js` that are a combination of `test-ci` and `e2e-ci` (respectively) + coverage configuration.